src/cmd/sanity_test/sanity_tests.go

```diff
@@ -152,8 +152,8 @@ func checkEmailRestoration(
 		"restore_folder_name", folderName)
 
 	childFolder, err := client.
-		UsersById(testUser).
-		MailFoldersById(folderID).
+		Users().ByUserId(testUser).
+		MailFolders().ByMailFolderId(folderID).
```

Comment:
I'm on the fence about keeping two items in the same line. Let me know if someone hs any opinion on it.

Reply:
against.  It introduces confusion about when single-line vs multi-line func chain is appropriate throughout the entire codebase.  Particularly because it's not obvious whether the func chain is producing a key-value-tuple pattern in the way that we see with the two-values-per-line such as maps and clues additions.  With func chains it's more simple to do one-line or per-line, and never inbetween.

Reply:
Switched over! :D